Terminal.Gui/Drawing/LineCanvasCell.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Text;
+
+
+namespace Terminal.Gui {
+
+	/// <summary>
+	/// Represents a single row/column within the <see cref="LineCanvas"/>. Includes the glyph and the foreground/background colors.
+	/// </summary>
+	public class LineCanvasCell {
```

Comment:
I really hate how not having nested classes makes it harder to be terse.

What do you think of leaving this `Cell`? Note that this class may end up being part of ConsoleDriver (or a variant of this) when #2616 is addressed.

Reply:
Sure, I was thinking it could be confused with having a role in `TableView` but it wasn't a deep concern.  I agree `Cell` is fine as it has xmldoc clearly describing what it does.